scikit_learn_script_mode_local_training_and_serving/scikit_learn_script_mode_local_training_and_serving.py

```diff
@@ -16,40 +16,54 @@
 import os
 
 from sagemaker.sklearn import SKLearn
+import sagemaker
+import boto3
 from sklearn import datasets
+from sklearn.model_selection import train_test_split
+from sklearn.metrics import mean_squared_error
 
-DUMMY_IAM_ROLE = 'arn:aws:iam::111111111111:role/service-role/AmazonSageMaker-ExecutionRole-20200101T000001'
+local_mode = False
```

Comment:
Since this is a SageMaker Local repo, can you please modify the default to be `True`?